src/EventEmitter.native.js

```diff
@@ -0,0 +1,13 @@
+import {NativeEventEmitter} from 'react-native';
+
+/**
+ * Key command Event listener.
```

Comment:
```suggestion
 * Key command EventEmitter.
```

Reply:
Can we please move these files to a subFolder pattern?
```
EventEmitter
  index.web.js
  index.native.js
```

Reply:
done